sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/LatestFnTest.java

```diff
@@ -166,12 +166,6 @@ public void testExtractOutput() {
   }
 
   @Test
-  public void testExtractOutputDefaultAggregator() {
```

Comment:
I think this is testing the `CombineFn` -- and should just be renamed from `Aggregator` to `Accumulator`.